### DISCUSSION TITLE
Kick off Github Actions Workflow when preview URL is deployed?

### DISCUSSION THREAD

**Discussion by hhimanshu** [vercel/vercel#6118] [Help]
Hello Folks,

I read https://rauchg.com/2020/2019-in-review#testing-the-jamstack and wondering how can I achieve the same with vercel preview deployments?

I have a [pull request](https://github.com/hhimanshu/typescript-serverless-api-vercel/pull/1) where I want to kick off another set of `jest` tests which tests APIs based on the preview URL. For that, the preview URL should be ready before the workflow can be kicked.

Are there any examples of how I can wait for deployment using Github Actions and only when the preview URL is ready (meaning the app is deployed), I kick off automated tests using a workflow?

Thank you

**Comment by aleksandras-va** [comment]
https://github.com/vercel/vercel/discussions/4501 something like this?

**Reply by hhimanshu** [reply]
Thanks, yes, I think this is what I am looking for!
